rock.osdeps

```diff
@@ -411,3 +411,12 @@ gawk:
 
 bzip2:
     debian,ubuntu: libbz2-dev
+
+slam/pcl:
+    ubuntu:
+        default: libpcl-dev
+        '14.04': libpcl-1.7-all-dev
```

Comment:
@planthaber: do the buildserver has additional PPAs? we decided in rock to not support PPAs for core packages